src/ec/packages/ec-component-footer-standardised/demo/data.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable import/no-extraneous-dependencies, no-param-reassign */
+import specData from '@ecl/ec-specs-footer-standardised/demo/data';
+
+function formatLink(l) {
```

Comment:
please check the new set of utilities for transforming the input data @planctus just created. Therefore you can use these new utilities instead. check adapted code for the footer-core as example:
https://github.com/ec-europa/ecl-twig/pull/185/commits/84df2df3eba94c73f1c1a918455fb88e06f9d725